spec/metadata_drop_spec.rb

```diff
@@ -2,10 +2,26 @@
 
 RSpec.describe(Jekyll::GitHubMetadata::MetadataDrop) do
   let(:overrides) { { "repository" => "jekyll/another-repo" } }
-  let(:config) { Jekyll::Configuration::DEFAULTS.merge(overrides) }
+  let(:config) { Jekyll::Configuration.from(overrides) }
   let(:site) { Jekyll::Site.new config }
   subject { described_class.new(site) }
 
+  context "in Liquid" do
+    before(:each) do
+      stub_all_api_requests
+      site.config.delete("repository")
+      site.config["github"] = subject
+    end
+
+    it "renders as-is in Liquid" do
+      payload = site.site_payload
+      expect(payload["site"]["github"]).to be_instance_of(described_class)
+      template = Liquid::Template.parse("{{ site.github }}")
+      result = template.render!(payload, :registers => { :site => site })
+      expect(result).to eql(subject.to_s)
```

Comment:
Aren't you just testing that Liquid calls `to_s` on this? Wouldn't this be true if the `to_s` method wasn't defined? Perhaps match a regex on the content?